bridges/AutoJMBridge.php

```diff
@@ -75,71 +36,102 @@ public function getName() {
 
 	public function collectData() {
 
-		$model_url = self::URI . $this->getInput('url');
-
-		// Build the GET data
-		$get_data = 'form[energy]=' . $this->getInput('energy') .
-			'&form[transmission]=' . $this->getInput('transmission') .
-			'&form[priceMin]=' . $this->getInput('priceMin') .
-			'&form[priceMin]=' . $this->getInput('priceMin');
+		// Get the number of result for this search
+		$search_url = self::URI . $this->getInput('url') . '&open=energy&onlyFilters=false';
 
 		// Set the header 'X-Requested-With' like the website does it
 		$header = array(
 			'X-Requested-With: XMLHttpRequest'
 		);
 
 		// Get the JSON content of the form
-		$json = getContents($model_url . '?' . $get_data, $header)
+		$json = getContents($search_url, $header)
 			or returnServerError('Could not request AutoJM.');
```

Comment:
please remove returnServerError here. It is useless, since getContents raises exception on error

Reply:
As requested, removed in 92a90ff :)